mmd_tools/externals/opencc/__init__.py

```diff
@@ -1,3 +1,5 @@
+# Copyright (C) 2016 Yichen Huang (Eugene)
```

Comment:
Since it is written in [LICENSE](https://github.com/MMD-Blender/blender_mmd_tools/blob/main/mmd_tools/externals/opencc/LICENSE) and [README.txt](https://github.com/MMD-Blender/blender_mmd_tools/blob/main/mmd_tools/externals/opencc/README.txt), why do we need to write here?